Failed to delete the temporary minecraft directory.

Hi,

I installed minerl according to the installation guide in the documentation today.
When I ran the Hello World example in the docs Minecraft was recompiled and launched but shortly after the Minecraft window closed and the message `Failed to delete the temporary minecraft directory.` was displayed.

I searched the internet but couldn't find anything helpful.

**My code:**
```
import minerl
import gym
import logging

logging.basicConfig(level=logging.DEBUG)


def main():
    # do your main minerl code
    env = gym.make('MineRLObtainDiamond-v0')


if __name__ == '__main__':
    main()
```

**Output:**
```
"C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Scripts\python.exe" "C:/Users/Simon Schmid/dev/Minecraft/MineRL/test.py"
C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\lib\site-packages\gym\logger.py:30: UserWarning: WARN: Box bound precision lowered by casting to float32
  warnings.warn(colorize('%s: %s'%('WARN', msg % args), 'yellow'))
INFO:minerl.env.malmo.instance.34cb6e:Starting Minecraft process: ['C:\\Users\\SIMONS~1\\AppData\\Local\\Temp\\tmpihp4w7wk\\Minecraft\\launchClient.bat', '-port', '10021', '-env', '-runDir', 'C:\\Users\\SIMONS~1\\AppData\\Local\\Temp\\tmpihp4w7wk\\Minecraft\\run']
INFO:minerl.env.malmo.instance.34cb6e:Starting process watcher for process 3316 @ localhost:10021
C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\lib\site-packages\gym\logger.py:30: UserWarning: WARN: Box bound precision lowered by casting to float32
  warnings.warn(colorize('%s: %s'%('WARN', msg % args), 'yellow'))
DEBUG:minerl.env.malmo.instance.34cb6e:This mapping 'snapshot_20161220' was designed for MC 1.11! Use at your own peril.
DEBUG:minerl.env.malmo.instance.34cb6e:#################################################
DEBUG:minerl.env.malmo.instance.34cb6e:         ForgeGradle 2.2-SNAPSHOT-3966cea        
DEBUG:minerl.env.malmo.instance.34cb6e:  https://github.com/MinecraftForge/ForgeGradle  
DEBUG:minerl.env.malmo.instance.34cb6e:#################################################
DEBUG:minerl.env.malmo.instance.34cb6e:               Powered by MCP unknown               
DEBUG:minerl.env.malmo.instance.34cb6e:             http://modcoderpack.com             
DEBUG:minerl.env.malmo.instance.34cb6e:         by: Searge, ProfMobius, Fesh0r,         
DEBUG:minerl.env.malmo.instance.34cb6e:         R4wk, ZeuX, IngisKahn, bspkrs           
DEBUG:minerl.env.malmo.instance.34cb6e:#################################################
DEBUG:minerl.env.malmo.instance.34cb6e:Found AccessTransformer: malmomod_at.cfg
DEBUG:minerl.env.malmo.instance.34cb6e::deobfCompileDummyTask
DEBUG:minerl.env.malmo.instance.34cb6e::deobfProvidedDummyTask
DEBUG:minerl.env.malmo.instance.34cb6e::getVersionJson
DEBUG:minerl.env.malmo.instance.34cb6e::extractUserdev UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::downloadClient SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::downloadServer SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::splitServerJar SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::mergeJars SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::applyBinaryPatches SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::extractDependencyATs SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::extractMcpData SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::extractMcpMappings SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::genSrgs SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::deobfMcMCP
DEBUG:minerl.env.malmo.instance.34cb6e:Applying SpecialSource...
DEBUG:minerl.env.malmo.instance.34cb6e:Applying Exceptor...
DEBUG:minerl.env.malmo.instance.34cb6e:Applying RuntimeInvisibleParameterAnnotations workaround...
DEBUG:minerl.env.malmo.instance.34cb6e::sourceApiJava UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::compileApiJava UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::processApiResources UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::apiClasses UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::copyModToClient UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::copyModToServer UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::copySrg UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::deleteSchemas
DEBUG:minerl.env.malmo.instance.34cb6e::copySchemas
DEBUG:minerl.env.malmo.instance.34cb6e::jaxb UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::sourceMainJava UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::compileJavawarning: [options] bootstrap class path not set in conjunction with -source 1.6
DEBUG:minerl.env.malmo.instance.34cb6e:Note: SpongePowered MIXIN Annotation Processor Version=0.7.5
DEBUG:minerl.env.malmo.instance.34cb6e:Note: ObfuscationServiceMCP supports type: "searge"
DEBUG:minerl.env.malmo.instance.34cb6e:Note: ObfuscationServiceMCP supports type: "notch"
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Loading searge mappings from C:\Users\Simon Schmid\.gradle\caches\minecraft\de\oceanlabs\mcp\mcp_snapshot\20161220\1.11.2\srgs\mcp-srg.srg
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Loading notch mappings from C:\Users\Simon Schmid\.gradle\caches\minecraft\de\oceanlabs\mcp\mcp_snapshot\20161220\1.11.2\srgs\mcp-notch.srg
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing refmap to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\compileJava-refmap.json
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing refmap to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\compileJava-refmap.json
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing searge output SRGs to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\mcp-srg.srg
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing notch output SRGs to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\mcp-notch.srg
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing refmap to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\compileJava-refmap.json
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing refmap to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\compileJava-refmap.json
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing searge output SRGs to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\mcp-srg.srg
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Writing notch output SRGs to C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\tmp\compileJava\mcp-notch.srg
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Some input files use or override a deprecated API.
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Recompile with -Xlint:deprecation for details.
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Some input files use unchecked or unsafe operations.
DEBUG:minerl.env.malmo.instance.34cb6e:Note: Recompile with -Xlint:unchecked for details.
DEBUG:minerl.env.malmo.instance.34cb6e:
DEBUG:minerl.env.malmo.instance.34cb6e::processResources UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::classes
DEBUG:minerl.env.malmo.instance.34cb6e::jar UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::extractNatives SKIPPED
DEBUG:minerl.env.malmo.instance.34cb6e::getAssetIndex UP-TO-DATE
DEBUG:minerl.env.malmo.instance.34cb6e::getAssets
DEBUG:minerl.env.malmo.instance.34cb6e::makeStart
DEBUG:minerl.env.malmo.instance.34cb6e::runClient
DEBUG:minerl.env.malmo.instance.34cb6e:2020-08-12 20:50:47,217 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Extra: []
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Found and added coremod: com.microsoft.Malmo.OverclockingPlugin
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Running with arguments: [--userProperties, {}, --assetsDir, C:/Users/Simon Schmid/.gradle/caches/minecraft/assets, --assetIndex, 1.11, --accessToken{REDACTED}, --version, 1.11.2, --tweakClass, net.minecraftforge.fml.common.launcher.FMLTweaker, --tweakClass, net.minecraftforge.gradle.tweakers.CoremodTweaker]
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Using primary tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Loading tweak class name net.minecraftforge.gradle.tweakers.CoremodTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Forge Mod Loader version 13.20.0.2228 for Minecraft 1.11.2 loading
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_202, running on Windows 10:amd64:10.0, installed at C:\Program Files\Java\jdk1.8.0_202\jre
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Managed to load a deobfuscated Minecraft name- we are in a deobfuscated environment. Skipping runtime deobfuscation
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Found a command line coremod : com.microsoft.Malmo.OverclockingPlugin
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/WARN]: The coremod com.microsoft.Malmo.OverclockingPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
DEBUG:minerl.env.malmo.instance.34cb6e:2020-08-12 20:50:47,409 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
DEBUG:minerl.env.malmo.instance.34cb6e:2020-08-12 20:50:47,433 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.7.5 Source=file:/C:/Users/Simon%20Schmid/.gradle/caches/modules-2/files-2.1/org.spongepowered/mixin/0.7.5-SNAPSHOT/c989878008c2c5ff1a7c6491c4103d8faa221d19/mixin-0.7.5-SNAPSHOT.jar Service=LaunchWrapper Env=CLIENT
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Compatibility level set to JAVA_8
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Calling tweak class net.minecraftforge.gradle.tweakers.CoremodTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Injecting location in coremod net.minecraftforge.fml.relauncher.FMLCorePlugin
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Injecting location in coremod net.minecraftforge.classloading.FMLForgePlugin
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Injecting location in coremod com.microsoft.Malmo.OverclockingPlugin
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLInjectionAndSortingTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Loading tweak class name org.spongepowered.asm.mixin.EnvironmentStateTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLDeobfTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Loading tweak class name net.minecraftforge.gradle.tweakers.AccessTransformerTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLInjectionAndSortingTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLInjectionAndSortingTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/INFO]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModManager$FMLPluginWrapper
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:47] [main/ERROR]: The binary patch set is missing. Either you are in a development environment, or things are not going to work!
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/ERROR]: FML appears to be missing any signature data. This is not a good thing
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModManager$FMLPluginWrapper
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModManager$FMLPluginWrapper
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Calling tweak class org.spongepowered.asm.mixin.EnvironmentStateTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Initialised Mixin FML Remapper Adapter with net.minecraftforge.fml.common.asm.transformers.deobf.FMLDeobfuscatingRemapper@5987e932
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLDeobfTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Calling tweak class net.minecraftforge.gradle.tweakers.AccessTransformerTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Loading tweak class name net.minecraftforge.fml.common.launcher.TerminalTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Calling tweak class net.minecraftforge.fml.common.launcher.TerminalTweaker
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/WARN]: Reference map 'mixins.replaymod.refmap.json' for mixins.overclocking.malmomod.json could not be read. If this is a development environment you can ignore this message
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:transform:58]: MALMO: Attempting to transform MinecraftServer
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:overclockRenderer:129]: MALMO: Found Minecraft, attempting to transform it
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:overclockRenderer:135]: MALMO: Found Minecraft.runGameLoop() method, attempting to transform it
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:overclockRenderer:150]: MALMO: Hooked into call to Minecraft.updateDisplay()
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: A re-entrant transformer '$wrapper.com.microsoft.Malmo.OverclockingClassTransformer' was detected and will no longer process meta class data
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: Launching wrapped minecraft {net.minecraft.client.main.Main}
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:transform:58]: MALMO: Attempting to transform MinecraftServer
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:overclockRenderer:129]: MALMO: Found Minecraft, attempting to transform it
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:overclockRenderer:135]: MALMO: Found Minecraft.runGameLoop() method, attempting to transform it
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:48] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:overclockRenderer:150]: MALMO: Hooked into call to Minecraft.updateDisplay()
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:49] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:transform:42]: Transformed Name: net.minecraft.client.entity.EntityPlayerSP
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:49] [main/INFO]: [com.microsoft.Malmo.OverclockingClassTransformer:transform:42]: Transformed Name: net.minecraft.client.entity.AbstractClientPlayer
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:49] [Client thread/INFO]: Setting user: Player41
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:51] [Client thread/WARN]: Skipping bad option: lastServer:
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:51] [Client thread/INFO]: LWJGL Version: 2.9.4
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: [STDOUT]: MALMO: Attempting to transform MinecraftServer
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: [STDOUT]: MALMO: Found GlStateManager, attempting to transform it
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: [STDOUT]: MALMO: Found GlStateManager.bindTexture() method, attempting to transform it
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: [STDOUT]: MALMO: Hooked into call to GlStateManager.bindTexture()
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: MinecraftForge v13.20.0.2228 Initialized
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: Replaced 232 ore recipes
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: Found 0 mods from the command line. Injecting into mod discoverer
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:52] [Client thread/INFO]: Searching C:\Users\Simon Schmid\AppData\Local\Temp\tmpihp4w7wk\Minecraft\run\mods for mods
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:53] [Client thread/INFO]: Forge Mod Loader has identified 5 mods to load
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Attempting connection with missing mods [minecraft, mcp, FML, forge, malmomod] at CLIENT
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Attempting connection with missing mods [minecraft, mcp, FML, forge, malmomod] at SERVER
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Reloading ResourceManager: Default, FMLFileResourcePack:Forge Mod Loader, FMLFileResourcePack:Minecraft Forge, FMLFileResourcePack:Microsoft Malmo Mod
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/WARN]: ResourcePack: ignored non-lowercase namespace: MalmoMod in C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\libs\MalmoMod-0.37.0.jar
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/WARN]: ResourcePack: ignored non-lowercase namespace: MalmoMod in C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\libs\MalmoMod-0.37.0.jar
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/WARN]: ResourcePack: ignored non-lowercase namespace: MalmoMod in C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\libs\MalmoMod-0.37.0.jar
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Processing ObjectHolder annotations
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Found 444 ObjectHolder annotations
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Identifying ItemStackHolder annotations
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Found 0 ItemStackHolder annotations
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Applying holder lookups
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Holder lookups applied
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Applying holder lookups
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Holder lookups applied
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Applying holder lookups
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Holder lookups applied
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: Configured a dormant chunk cache size of 0
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Forge Version Check/INFO]: [forge] Starting version check at http://files.minecraftforge.net/maven/net/minecraftforge/forge/promotions_slim.json
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:54] [Client thread/INFO]: [STDOUT]: Testing schemas against internal version number: 0.37
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:55] [Client thread/INFO]: [STDOUT]: [LOGTOPY] Performance directory not specified.
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:55] [Client thread/INFO]: [STDOUT]: [ERROR] Seed specified was $seed. Expected a long (integer).
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:55] [Client thread/INFO]: Applying holder lookups
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:55] [Client thread/INFO]: Holder lookups applied
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:55] [Client thread/INFO]: Injecting itemstacks
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:55] [Client thread/INFO]: Itemstack injection complete
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:55] [Forge Version Check/INFO]: [forge] Found status: OUTDATED Target: 13.20.1.2386
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:57] [Sound Library Loader/INFO]: Starting up SoundSystem...
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:57] [Thread-5/INFO]: Initializing LWJGL OpenAL
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:57] [Thread-5/INFO]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:57] [Thread-5/INFO]: OpenAL initialized.
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:57] [Sound Library Loader/INFO]: Sound engine started
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:59] [Client thread/INFO]: Max texture size: 16384
DEBUG:minerl.env.malmo.instance.34cb6e:[20:50:59] [Client thread/INFO]: Created: 16x16 textures-atlas
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/INFO]: [STDOUT]: CLIENT request state: WAITING_FOR_MOD_READY
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/INFO]: Injecting itemstacks
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/INFO]: Itemstack injection complete
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/INFO]: Forge Mod Loader has successfully loaded 5 mods
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/INFO]: Reloading ResourceManager: Default, FMLFileResourcePack:Forge Mod Loader, FMLFileResourcePack:Minecraft Forge, FMLFileResourcePack:Microsoft Malmo Mod
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/WARN]: ResourcePack: ignored non-lowercase namespace: MalmoMod in C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\libs\MalmoMod-0.37.0.jar
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/WARN]: ResourcePack: ignored non-lowercase namespace: MalmoMod in C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\libs\MalmoMod-0.37.0.jar
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:00] [Client thread/WARN]: ResourcePack: ignored non-lowercase namespace: MalmoMod in C:\Users\Simon Schmid\dev\Minecraft\MineRL\venv\Lib\site-packages\minerl\env\Malmo\Minecraft\build\libs\MalmoMod-0.37.0.jar
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:02] [Client thread/INFO]: SoundSystem shutting down...
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:02] [Client thread/WARN]: Author: Paul Lamb, www.paulscode.com
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:02] [Sound Library Loader/INFO]: Starting up SoundSystem...
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:02] [Thread-7/INFO]: Initializing LWJGL OpenAL
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:02] [Thread-7/INFO]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:02] [Thread-7/INFO]: OpenAL initialized.
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:03] [Sound Library Loader/INFO]: Sound engine started
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:04] [Client thread/INFO]: Max texture size: 16384
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:04] [Client thread/INFO]: Created: 512x512 textures-atlas
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:05] [Client thread/WARN]: Skipping bad option: lastServer:
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:06] [Client thread/INFO]: [STDOUT]: CLIENT enter state: WAITING_FOR_MOD_READY
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:06] [Client thread/INFO]: [STDOUT]: ***** Start MalmoEnvServer on port 12900
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:06] [Client thread/INFO]: [STDOUT]: CLIENT request state: DORMANT
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:06] [Client thread/INFO]: [STDOUT]: CLIENT enter state: DORMANT
INFO:minerl.env.malmo.instance.34cb6e:Minecraft process ready
WARNING:minerl.env.malmo.instance.34cb6e:Tried to launch Minecraft on port 10021 but that port was taken, instead Minecraft is using port 12900.
INFO:minerl.env.malmo:Logging output of Minecraft to .\logs\mc_1021.log
Failed to delete the temporary minecraft directory.
DEBUG:minerl.env.malmo.instance.34cb6e:[20:51:07] [EnvServerSocketHandler/INFO]: Java has been asked to exit (code 0) by net.minecraftforge.fml.common.FMLCommonHandler.exitJava(FMLCommonHandler.java:659).

Process finished with exit code 0
```

--- a/minerl/env/malmo.py
+++ b/minerl/env/malmo.py
@@ -758,17 +758,17 @@ class MinecraftInstance(object):
             # Wait for the process to start.
             time.sleep(1)
             # kill the minecraft process and its subprocesses
-            try:
-                shutil.rmtree(self.instance_dir)
-            except:
-                print("Failed to delete the temporary minecraft directory.")
-
             if self._kill_minecraft_via_malmoenv(self.host, self.port):
                 # Let the minecraft process term on its own terms.
                 time.sleep(2)
 
             # Now lets try and end the process if anything is laying around
             minerl.utils.process_watcher.reap_process_and_children(self.minecraft_process)
+
+            try:
+                shutil.rmtree(self.instance_dir)
+            except:
+                print("Failed to delete the temporary minecraft directory.")
 
             if self in InstanceManager._instance_pool:
                 InstanceManager._instance_pool.remove(self)

